scripts/output/testing/main_data/Qatar.csv

```diff
@@ -1,589 +1,611 @@
+Qatar,2022-01-03,3197708,0.152,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-04,3203298,0.198,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-05,3210583,0.248,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-06,3217127,0.303,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-07,3223877,0.356,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-08,3231475,0.405,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-09,3237735,0.456,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-10,3242721,0.521,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-11,3247302,0.577,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-12,3251971,0.648,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-13,3255962,0.714,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-14,3260304,0.776,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-15,3264723,0.86,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-16,3267807,0.955,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-17,3270498,1.021,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-18,3273734,1.055,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-19,3276743,1.089,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-20,3279242,1.12,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-21,3281825,1.168,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-22,3284276,1.233,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-23,3287344,1.17,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-24,3290271,1.092,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
+Qatar,2022-01-25,3292852,1.05,people tested,https://covid19.moph.gov.qa/EN/Pages/default.aspx,Qatar Ministry of Public Health,
```

Comment:
I was baffled by the positive rates so I manually calculated it and, YEAH number of positive case was larger than the number of people tested!